README.md

```diff
@@ -14,12 +14,25 @@ More information is available on the
 
 ## Installation
 
+Prerequisites
+-------------
+* Have the most recent version of Java (Java 1.8)
```

Comment:
While Java 8 is recommended, it's not the most recent version of Java. Java SE 10 actually [just came out](http://www.oracle.com/technetwork/java/javase/overview/index.html). Because of this, I've only personally tested with Java 8 and Java 9.